src/include/txn_inline.h

```diff
@@ -777,11 +777,11 @@ __wt_txn_visible_all(WT_SESSION_IMPL *session, uint64_t id, wt_timestamp_t times
 }
 
 /*
- * __wt_txn_newest_visible_all --
+ * __wt_txn_has_newest_and_visible_all --
  *     Check whether a given newest time window is globally visible.
```

Comment:
Also update the description?

Reply:
yes sorry 79ce98943f51a8f76b97834239bcdfcfc765ca43